File: src/Bowl/Models/Api/Bowl.cs
using System;
using System.Collections.Generic;

namespace Bowl.Models.Api
{
    public class Bowl : Item
    {
        public Guid Guid { get; set; }
        public string Name { get; set; }
        public string Location { get; set; }
        public string FoodName { get; set; }
        public decimal FoodAmount { get; set; }
        public decimal FoodAlert { get; set; }
        public bool Open { get; set; }
        public bool Distributing { get; set; }

        //Many to Many
        //public ICollection<BowlMeal> BowlMeals { get; set; }
        public static object getJsonBowl(Bowl bowl) => new
        {
            bowl.Id,
            bowl.Guid,
            bowl.Name,
            bowl.Location,
            bowl.FoodName,
            bowl.FoodAmount,
            bowl.FoodAlert
        };
    }
}

File: src/Bowl/Scripts/Sources/bowls/bowl.ts
export class Bowl {

    constructor(obj?: any) {

        if (obj !== null) {
            for (var prop in obj) {

                if (obj[prop] !== null) {
                    this[prop] = obj[prop];
                }
            }
        }
    }

    public id: number = 0;
    public bowlId: number = 0;
    public name: string = "Not assigned";
    public location: string = "Not assigned";
    public foodName: string = "Not assigned";
    public foodAmount: number = 0.00;
    public foodAlert: number = 0.00;
    public open: boolean = false;
    public distributing: boolean = false;
}